AEPEdgeMedia.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "AEPEdgeMedia"
-  s.version          = "1.0.0-beta"
+  s.version          = "4.0.0"
   s.summary          = "Experience Platform Edge Media extension for Adobe Experience Platform Mobile SDK. Written and maintained by Adobe."
```

Comment:
Let's update summary and description once we get the final confimation on the email discussion.

Reply:
👍 
We can do that in another PR if needed if we don't get a decision soon.